### DISCUSSION TITLE
What is the correct way to update the yarn version with multiple users?

### DISCUSSION THREAD

**Discussion by ghost** [yarnpkg/berry#3885] [General]
If I upgrade yarn and my `.yarnrc.yml` now contains:

```yml
yarnPath: .yarn/releases/yarn-3.1.1.cjs
```

and I commit this, everyone else that now checks it out now has a broken version of yarn:

```sh
Error: Cannot find module '/.yarn/releases/yarn-3.1.1.cjs'
```

and has to do something like:

```sh
mv .yarnrc.yml .yarnrc.yml.tmp
yarn set version 3.1.1
rm .yarnrc.yml
mv .yarmrc.yml.tmp .yarnrc.yml
```

Is there a better way?

**Comment by merceyz** [comment]
You need to update your `.gitignore` file so that `.yarn/releases` isn't ignored
https://yarnpkg.com/getting-started/qa/#which-files-should-be-gitignored
